src/App.jsx

```diff
@@ -5,33 +5,29 @@ import DraftCard from './Components/DraftCard'
 import Card from './Components/Card'
 
 
-
 function App() {
 
   const [cards, setCards] = useState([]);
-  // const [doneCardsTotal, setDoneCardsTotal] = useState(0);
   const [maxIndexKey, setMaxIndexKey] = useState(0);
-  
+  const [selectedCardID, setSelectedCardID] = useState(0);
```

Comment:
you might want to change the initial value from 0 to null. 0 potentially represents a card with id of 0, while null would represent no card and I think that would be more accurate in this case

Reply:
Makes sense , didn't know you could set null in use state.